test/modules/TestAnalyzer/AggregateCloudOperationReport.cs

```diff
@@ -33,7 +33,9 @@ public AggregateCloudOperationReport(string moduleId, IDictionary<string, Tuple<
 
         public IDictionary<string, string> TestInfo { get; }
 
-        public bool IsPassed => this.StatusCodes.All(x => x.StatusCode.StartsWith("200") || x.StatusCode.Equals("OK"));
+        public bool IsPassed => this.StatusCodes.All(x => x.StatusCode.StartsWith("200")
+                                                        || x.StatusCode.Equals("OK")
+                                                        || x.StatusCode.Equals("NotFound"));
```

Comment:
It looks weird to me, why "NotFound" still make the report passed?

Reply:
I agree that this looks weird, but this is a byproduct of how we defined the status code on DeviceNotFoundException. We could change it to a custom status code?

Reply:
Forget about current implementation.  For connectivity, I expect it should report what the result is to TRC.  For LH, if there is any exception during DM invoke call, we should eat up the exception, log it and retry (without increasing sequence number [optional if this doesn't impact to Connectivity test]).  If that's the case, we don't need to make change here.

Reply:
To implement this, you can keep the change in DMSenderBase.  You may need to create a new subclass from DMSenderBase for LH DM Cloud Sender.

Reply:
I implemented it like this in order to keep the test-specific (longhaul vs connectivity) in the module that defines the test type (Analyzer vs TRC). Although I forgot that the TestAnalyzer expects the module to send test results rather than response codes. I think you are right and will make the change.